jupyter_book/commands/__init__.py

```diff
@@ -29,23 +32,28 @@ def main():
 @click.option("--toc", default=None, help="Path to the Table of Contents YAML file")
 @click.option("-W", "--warningiserror", is_flag=True, help="Error on warnings.")
 @click.option(
-    "--build",
+    "--builder",
     default="html",
-    help="What kind of output to build. Must be one of {BUILD_OPTIONS}",
+    help="Which Sphinx Builder to use. Must be one of {BUILDER_OPTIONS}",
```

Comment:
I think we should try to avoid using the word "Sphinx" in top-level user documentation if possible..I think only "power users" should need to see the word Sphinx. What do you think?

```suggestion
    help="Which builder to use. Must be one of {BUILDER_OPTIONS}",
```